src/Tile/Tile.js

```diff
@@ -34,7 +37,20 @@ const Tile = props => {
         <div
             {...rest}
             className={tileClasses}>
-            {children}
+            {productTile &&
+                <div className='fd-product-tile__media' style={{ backgroundImage: 'url(' + backgroundImage + ')' }} />
+            }
```

Comment:
I created a conditional for product tile rather than if the background image exists bc product tiles should have background images with them always. This will eliminate the issue Jenna mentioned with background image rendering undefined for tiles that are not product.

Reply:
Well, _technically_ it won't.  There are really two props needed for a product tile:  `productTile` and `backgroundImage` so it's possible the consumer passes one, but not the other.  That said, this has become more of a pattern-based component so I think this is a good change.  We _could_ make `productTile` a shape prop with a required `backgroundImage`, but that might be overkill here.  As we've said in this PR, the consumer just wants to know which props to pass and the example page will help them with that.